### PR TITLE
Anchors hint

### PR DESCRIPTION
**Pull request by MarcinAman**
Basically anchors should right now work in the same way they did before

### CODE REVIEW

File: plugins/base/src/main/resources/dokka/scripts/platformContentHandler.js [hunk 1]
```diff
@@ -42,15 +42,14 @@ function handleAnchor() {
...
-                let found = document.querySelector('.tabs-section > .section-tab[data-togglable="' + tab.getAttribute("data-togglable") + '"]')
                 toggleSections(tab)
-                const content = element.nextElementSibling
-                if(content){
-                    content.classList.add('anchor-highlight')
-                    highlightedAnchor = content
-                }
-                element.scrollIntoView({behavior: "smooth"})
             }
+            const content = element.nextElementSibling
```
Comment (MarcinAman) [inline plugins/base/src/main/resources/dokka/scripts/platformContentHandler.js:47]: This change is unrelated to anchors rendering, but I saw that anchors are not highlighted correctly when there is no tab (eg. on module page)

**Review by Kordyjan — CHANGES_REQUESTED** [review]
In my opinion this pr should add potential anchors for all table rows. Of course, I'm open for discussion.

**Comment by MarcinAman** [comment]
> In my opinion this pr should add potential anchors for all table rows. Of course, I'm open for discussion.

I thought about it and i don't see a reason why we should preemptively generate anchors. Also this is super bad in current state since anchors are not unique (see https://github.com/Kotlin/dokka/issues/1258 and anchors don't have a sourceset in them so every sourceset tab would be broken). Using an extra is a decent idea since it gives us control over what we want to link to and introduces a straight forward logic how to do it.

**Review by BarkingBad — COMMENTED** [review]
I think currently we currently generate anchors for every element possible (I think I added that to handle typealiases and properties (which will have their own pages since #1494)) so we should check that PR for any possible inconvenient behaviour.

**Review by kamildoleglo — APPROVED** [review]

**Review by kamildoleglo — APPROVED** [review]

**Pull request merged**